### ISSUE TITLE
Handling of errors on initial manifest import request

### ISSUE THREAD

**Issue by wabain** [DDMAL/misirlou#54]
As of 722a26f97127718dbbf1d61a7b3f0c010f43f507 the manifest upload process synchronously dereferences the URL posted for upload and responds to a failure to get URLs by returning a 200 with a body like:

``` json
{ "error": "Could not find manifests", "status": -1 }
```

This is roughly the format of the responses from the /status endpoint, but it's different from the response an upload request gets if URLs were located, which uses `status` for a different purpose:

``` json
{ "status": "http://localhost:8001/status/a6579f8b-3e1e-49a5-821b-5ceee0149564/" }
```

I think it could return a 400-level response and drop the status field.

**Comment by agpar** [comment]
Agree 100%, will implement this tomorrow afternoon.

**Comment by agpar** [comment]
Fixed in 76d119e09365248cb44a5fbaea53a0d6bc6dcbfc
